AppCenter/AppCenterTests/MSUtilityTests.m

```diff
@@ -894,4 +897,36 @@ - (void)testObfuscateRedirectUri {
   XCTAssertFalse([obfuscatedString rangeOfString:kMSRedirectUriObfuscatedTemplate].location == NSNotFound);
 }
 
+- (void)testPerformSelectorOnMainThread {
```

Comment:
Is this simulating a call from main thread or from background thread? We need to test both cases.